money.go

```diff
@@ -89,7 +89,7 @@ func New(amount int64, code string) *Money {
 // NewFromFloat creates and returns new instance of Money from a float64.
 // Always rounding trailing decimals down.
 func NewFromFloat(amount float64, currency string) *Money {
-	currencyDecimals := math.Pow10(GetCurrency(currency).Fraction)
+	currencyDecimals := math.Pow10(newCurrency(currency).get().Fraction)
```

Comment:
What do you think on letting GetCurrency to return currency in a case insensitive manner?

`newCurrecy().get()` seems a little convolute to do the same.

Reply:
Thanks, that's a good one. (in my way , GetCurrency still fails with uppercase codes.)
Fixed GetCurrency to be case-insensitive